pkg/thing/interactors/register_thing.go

```diff
@@ -82,25 +82,27 @@ func (i *ThingInteractor) Register(authorization, id, name string) error {
 	if err != nil {
 		errReply := i.reply(id, "", err)
 		if errReply != nil {
-			i.logger.Error(errReply)
+			i.logger.Debug(err)
```

Comment:
```suggestion
			i.logger.Error(err)
```

Reply:
This log won't be showed if the log level is INFO or ERROR just if it is DEBUG

Reply:
In order to add some context to the errors, I've made some changes, including removing the logging in this interactor function. Done!